cypress/integration/user-control-panel.js

```diff
@@ -0,0 +1,42 @@
+if (Cypress.env('API') !== 'guillotina') {
+  describe('User Control Panel Test', () => {
+    beforeEach(() => {
+      // given a logged in editor
+      // and a folder that contains a document
+      // and the folder contents view
+      cy.visit('/');
+      cy.autologin();
+      cy.visit('/controlpanel/users');
+      cy.waitForResourceToLoad('@navigation');
+      cy.waitForResourceToLoad('@breadcrumbs');
+      cy.waitForResourceToLoad('@actions');
+      cy.waitForResourceToLoad('@types');
+    });
+
+    it.only('Should add User to controlPanel', () => {
+      // when I added a user from controlPanel
+      cy.get('.addSVG').click();
```

Comment:
We might want to add a more meaningful class to the code here so that we can use a more meaningful selector in the test.